examples/job.yml

```diff
@@ -1,18 +1,24 @@
 - job: new_backend
     timeoutInMinutes: 90
     steps:
-    - checkout: self 
+    - checkout: self
       persistCredentials: true
       clean: true
 
+    - task: InstallSSHKey@0
```

Comment:
Can we add some comments to this `.yaml` file to explain why we need to add this deployment key.

Reply:
```
# This task enables ssh authentication.
# It keeps git permissions even after long build time.
```